huaweicloud/services/as/resource_huaweicloud_as_group.go

```diff
@@ -741,9 +709,8 @@ func resourceASGroupDelete(ctx context.Context, d *schema.ResourceData, meta int
 		return diag.Errorf("error creating autoscaling client: %s", err)
 	}
 
-	timeout := d.Timeout(schema.TimeoutDelete)
 	groupID := d.Id()
-	logp.Printf("[DEBUG] Begin to delete AS group %s", groupID)
+	timeout := d.Timeout(schema.TimeoutDelete)
```

Comment:
timout is used only once. we don't need to define a variable.

Reply:
I think we can keep it as is